src/app/App.js

```diff
@@ -54,7 +57,14 @@ export default class App extends Component {
   }
 
   filterMovies = (query) => {
-    const filteredList = this.state.movies.filter(movie => movie.title.toUpperCase().includes(query.toUpperCase()))
+    const formattedQuery = query.toUpperCase();
+
+    const filteredList = this.state.movies.filter(movie => {
+    return movie.title.toUpperCase().includes(formattedQuery)
+    || movie.genres.toString().toUpperCase().includes(formattedQuery)
+    || movie.overview.toUpperCase().includes(formattedQuery)
+  });
```

Comment:
Good refactor here - added search bar capability but the code is a lot more readable.